src/TodoListItem.module.css

```diff
@@ -0,0 +1,61 @@
+.ListItem {
+    color: #00396a; 
+  }
+  
+  .ListItemCompleted {
+    color: rgb(0, 87, 128);
+    text-decoration: line-through;
+  }
+  
+  .ListItemNotCompleted {
+    color: rgb(101, 204, 245);
```

Comment:
Once I got your css working properly I found out this color needs to be changed to something with more contrast as it's currently barely readable.